pytorch_lightning/trainer/optimizers.py

```diff
@@ -94,6 +94,7 @@ def configure_schedulers(self, schedulers: list, monitor: Optional[str] = None):
         lr_schedulers = []
         default_config = {
             'scheduler': None,
+            'name': None,  # no custom name
```

Comment:
is this needed as with `my_dictionary.get('missing-key')` returns None by default

Reply:
Yes because we use this `default_config` dict to validate the user input just below.